bokehjs/src/lib/models/tools/inspectors/hover_tool.ts

```diff
@@ -397,7 +397,7 @@ export class HoverToolView extends InspectToolView {
 
         let cell: HTMLElement
 
-        cell = div({style: {display: "table-cell"}, class: 'bk-tooltip-row-label'}, `${label}: `)
+        cell = div({style: {display: "table-cell"}, class: 'bk-tooltip-row-label'}, label.length != 0 ? "${label}: " : "")
```

Comment:
Actually my suggestion in the issue was slightly wrong. It should be ``${label}: `` (backticks instead of quotes, the same as above).

Reply:
I was already wondering about that... I gotta admit that I didn't try it out myself because setting up the dev environment seemed to be more effort than the proverbial `pip install .`.

Reply:
Fixed now.

Reply:
> (...) setting up the dev environment seemed to be more effort than the proverbial pip install 

Setting up a dev environment is indeed a pain in the bottom currently.